subsys/net/lib/sockets/Kconfig

```diff
@@ -62,6 +62,12 @@ config NET_SOCKETS_TLS_MAX_CIPHERSUITES
 	  By default, all ciphersuites that are available in the system are
 	  available to the socket.
 
+config NET_SOCKETS_OFFLOAD
+	bool "Offload Socket APIs"
```

Comment:
Add [EXPERIMENTAL] at the end of the description

Reply:
Agreed.